ui/component/upcomingClaims/view.jsx

```diff
@@ -130,15 +171,12 @@ const ScheduledStreams = (props: Props & StateProps & DispatchProps) => {
             button="link"
             iconRight={ICONS.ARROW_RIGHT}
             className="claim-grid__title--secondary"
-            onClick={() => {
-              window.scrollTo({ top: 0, left: 0, behavior: 'smooth' });
-              setShowAllUpcoming(false);
-            }}
```

Comment:
In mobile, we get disoriented when trying to collapse.

I understand the original code wouldn't work 100% if the section is not at the top either, but I guess better than being in a random place?

<img width="353" alt="image" src="https://github.com/OdyseeTeam/odysee-frontend/assets/64950861/903f740b-702e-42f2-8243-6ce6e13e3ab8">